### PR TITLE
docs(react-menu): fix stories

### PR DESCRIPTION
**Pull request by layershifter**
See comments on files 👇

### CODE REVIEW

File: packages/react-components/react-menu/src/stories/MenuControlledCheckboxItems.stories.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const PasteIcon = bundleIcon(ClipboardPasteFilled, ClipboardPasteRegular);
 const EditIcon = bundleIcon(EditFilled, EditRegular);
 
 export const ControlledCheckboxItems = () => {
-  const [checkedValues, setCheckedValues] = React.useState<Record<string, string[]>>({});
+  const [checkedValues, setCheckedValues] = React.useState<Record<string, string[]>>({ edit: ['cut', 'paste'] });
```
Comment (layershifter) [inline packages/react-components/react-menu/src/stories/MenuControlledCheckboxItems.stories.tsx:21]: Added default values to make it more obvious.

File: packages/react-components/react-menu/src/stories/MenuControlledRadioItems.stories.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const PasteIcon = bundleIcon(ClipboardPasteFilled, ClipboardPasteRegular);
 const EditIcon = bundleIcon(EditFilled, EditRegular);
 
 export const ControlledRadioItems = () => {
-  const [checkedValues, setCheckedValues] = React.useState<Record<string, string[]>>({ checkbox: ['2'] });
+  const [checkedValues, setCheckedValues] = React.useState<Record<string, string[]>>({ font: ['calibri'] });
```
Comment (layershifter) [inline packages/react-components/react-menu/src/stories/MenuControlledRadioItems.stories.tsx:21]: Oops, probably was overlooked during refactors.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `181.156 kB`<br />`50.603 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.078 kB`<br />`11.098 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `111.638 kB`<br />`34.054 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `114.813 kB`<br />`34.527 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/303947a3e3687058c902a9a8bee1ecd1f19a92c8'>303947a3e3687058c902a9a8bee1ecd1f19a92c8</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/Avatar-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/Avatar-mount.html">1062</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/Button-mount_base.html">664</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/Button-mount.html">656</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProvider-mount_base.html">2198</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProvider-mount.html">2165</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProviderWithTheme-mount_base.html">347</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProviderWithTheme-mount.html">329</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">276</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">278</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">367</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/MakeStyles-mount_base.html">1878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23225/perf-test-react-components/MakeStyles-mount.html">1852</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Pull request merged**